src/_playground/documentation/Playground/Playground.js

```diff
@@ -533,9 +533,9 @@ class Playground extends Component {
               } else if (this.state.component === 'product') {
                   componentToGenerate = (
                       <Tile>
-                          <ProductTile>
-                              <ProductTile.Media image='https://techne.yaas.io/images/product-thumbnail-wide.png' />
-                              <ProductTile.Content
+                          <Tile>
```

Comment:
I think we should either remove the playground component for this page or fix it to work with the new changes so that all the options are there.